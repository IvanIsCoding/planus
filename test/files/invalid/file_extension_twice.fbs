file_extension "FOO";
file_extension "BAR";