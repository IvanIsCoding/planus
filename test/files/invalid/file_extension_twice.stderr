[0m[1m[38;5;9merror[0m[1m: Cannot set file_extension twice[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/file_extension_twice.fbs:1:1
  [0m[34mâ”‚[0m
[0m[34m1[0m [0m[34mâ”‚[0m [0m[31mfile_extension "FOO";[0m
  [0m[34mâ”‚[0m [0m[31m^^^^^^^^^^^^^^^^^^^^^[0m [0m[31mfirst declaration was here[0m
[0m[34m2[0m [0m[34mâ”‚[0m [0m[31mfile_extension "BAR";[0m
  [0m[34mâ”‚[0m [0m[31m^^^^^^^^^^^^^^^^^^^^^[0m [0m[31madditional declaration was here[0m

Error: Bailing because of errors
